src/azure-cli/azure/cli/command_modules/resource/commands.py

```diff
@@ -387,6 +393,13 @@ def load_command_table(self, _):
         g.custom_show_command('show', 'get_policy_setdefinition')
         g.custom_command('update', 'update_policy_setdefinition')
 
+    with self.command_group('policy exemption', resource_policy_exemptions_sdk, resource_type=ResourceType.MGMT_RESOURCE_POLICY, min_api='2020-09-01') as g:
+        g.custom_command('create', 'create_policy_exemption')
+        g.custom_command('delete', 'delete_policy_exemption')
```

Comment:
please add confirmation=True for delete command

Reply:
Existing policy command don't ask for confirmation. I'd keep the consistency on policy related commands.